src/Typist.jsx

```diff
@@ -44,6 +45,7 @@ export default class Typist extends Component {
   state = {
     text: [],
     isDone: false,
+    delay: 0,
```

Comment:
I think we can keep this as an instance variable instead of state given that we don't really need to trigger a re-render as we keep track of this value